cherrypy/test/test_config.py

```diff
@@ -221,8 +221,9 @@ def testHandlerToolConfigOverride(self):
         # the favicon in the page handler to be '../favicon.ico',
         # but then overrode it in config to be './static/dirback.jpg'.
         self.getPage('/favicon.ico')
-        self.assertBody(open(os.path.join(localDir, 'static/dirback.jpg'),
-                             'rb').read())
+        testfile = open(os.path.join(localDir, 'static/dirback.jpg'),'rb')
```

Comment:
This should use a CM (`with`) in order to guarantee that the file descriptor gets closed always.